src/syntax/functionParameterConfusion/FunctionParameterConfusion.kts

```diff
@@ -0,0 +1,11 @@
+package syntax.functionParameterConfusion
+
+val items = listOf("a", null, "b")
+
+println(items.joinToString { " + " })
+
+// What will the output be?
+// a) null
```

Comment:
This is a bit strange, what would be the reasoning to get this output? Maybe "a null b" (i.e. no separators) would be a better non-answer.

Reply:
My thinking here is that `" + "` might look like "use the plus operator" to concatenate.

Reply:
Thanks for the feedback! I have removed `null` now and added your option of the result string not having any separator.